core/templates/pages/story-viewer-page/story-viewer-page-root.component.html

```diff
@@ -19,3 +19,291 @@
 
   </oppia-base-content>
 </oppia-angular-root>
+
+<style>
```

Comment:
Just to check -- I would have expected these styles to be deleted from somewhere else, so as not to duplicate code. Can you talk more about the approach here?

Reply:
These styles were deleted in PR #13533 from the `story-viewer-page.mainpage.html` file (the file was also deleted in that PR) but I forgot to put the styles somewhere else. Migrating, pages to the angular router was a repetitive task and all pages involved deleting `*mainpage.html` files. Maybe i deleted the `story-viewer-page.mainpage.html` without looking what's inside carefully.